docs/pages/helm/guides/ownconfig.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: Running a Teleport cluster with a custom Teleport configuration in Kubernetes using Helm
+description: How to install and configure a Gravitational Teleport cluster with a custom config in Kubernetes using Helm
+---
+
+In this guide, we'll go through how to set up a Teleport cluster in Kubernetes using a custom [`teleport.yaml`](../../config-reference.mdx)
+config file using Teleport Helm charts.
+
+This setup can be useful when you already have an existing Teleport cluster and would like to start running it in Kubernetes, or when
+migrating your setup from an older version of the Helm charts.
+
+We also have a specific [migration guide from older versions of the Helm charts](./migration.mdx) which may also be useful.
+
+We'll assume that you've finished [the Getting Started guide](./getting-started.mdx) and have Helm configured successfully.
+
+If not, please finish that guide first before starting this one!
+
+### Setting up a Teleport cluster with Helm using a custom config
+
+In `ownconfig` mode, the `teleport-cluster` Helm chart does not create a `ConfigMap` containing a `teleport.yaml` file for you, but
+expects that you will provide this yourself.
+
+For this example, we'll be using this `teleport.yaml` configuration file:
+
+```yaml
+teleport:
+  log:
+    output: stderr
+    severity: INFO
+
+auth_service:
+  enabled: true
+  cluster_name: ownconfig.example.com
+  tokens:
+  - "proxy,node:cars-are-nicer-than-trains"
```

Comment:
this will end up all over the internet, let's use some `cat > $(uuid)` magic so readers will generate random IDs